### ISSUE TITLE
[ENHANCEMENT] Padding adjustment if the comment begins with a quote

### ISSUE THREAD

**Issue by bohrabhijeet** [laurencedawson/sync-for-reddit#1130]
**Is your feature request related to a problem? Please describe.**
There's almost no padding between a quote and the username if the comment starts with a quote

**Comparison**

- Reddit
![Screenshot_20220423-181730~2.jpg](https://user-images.githubusercontent.com/34155181/164895210-2c390a2e-f649-44f8-8a93-fdd4eb11ae06.jpg)

- Sync
![Screenshot_20220423-181326~3.jpg](https://user-images.githubusercontent.com/34155181/164895220-91dc887e-5337-4f97-bfb7-2a63e92aaaed.jpg)

**Comment by laurencedawson** [comment]
Quotes imo should always indent.

**Comment by bohrabhijeet** [comment]
> Quotes imo should always indent.

Yes, agreed. But it's too close to the username. There should be _some_ padding between the **username** and the indent if the comment begins with a quote.

**Comment by laurencedawson** [comment]
Gotcha
